src/chart/tree/TreeView.js

```diff
@@ -161,6 +161,8 @@ export default echarts.extendChartView({
         var max = [];
         bbox.fromPoints(points, min, max);
 
+        // Store min max because when collapse the root node after roam,
+        // the root node will disappear.
```

Comment:
This comment might be ambiguous for the reader.
Probably we should describe it like: if we dont store min/max, the root will disappear.